client/src/pages/Login.js

```diff
@@ -1,38 +1,84 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
+import Modal from "react-bootstrap/Modal";
 
```

Comment:
Hi Christian, the code looks good, but will you please maybe see to insert the template so that the navbar and footer could reflect on this page as well as the same with the login page